testsuite/integration-arquillian/tests/base/src/test/resources/arquillian.xml

```diff
@@ -74,6 +74,7 @@
                 ${auth.server.feature}
             </property>
             <property name="javaVmArguments">
+                ${auth.server.jboss.jvm.args}
```

Comment:
maybe naming could reflect this parameter is used for debug, e.g. auth.server.jboss.jvm.debug.args ?